static/js/around-map.js

```diff
@@ -110,11 +107,15 @@ function findPoi (id) {
 }
 
 function highlight (poi_id) {
-  const cardActive = document.getElementsByClassName('current-card')
+  const cardActive = document.getElementsByClassName('current-card')[0]
```

Comment:
`document.querySelector('.current-card')` will take the first matching that class.